### ISSUE TITLE
[1.7.10] Gasstation problem with Biome decoration fix which disables the mod patch 

### ISSUE THREAD

**Issue by quentin452** [quentin452/1.7.10-Forge-renewed-v1#75]
If i use spongemixin + mixingasm i don't get theses warns
biome decoration fix : https://www.curseforge.com/minecraft/mc-mods/biome-decoration-fix/

descriptions : 

when i use gasstation with biome decoration fix , i have warnings , the biome decoration fix patch is not applied and so i can still have the already decorating crash

mods list : 

00gasstation-mc1.7.10-0.3.5
alreadydecoratingfix-1.0a

crash(exemple) : 

[crash-2022-10-08_20.55.06-server.txt](https://github.com/FalsePattern/GasStation/files/9742439/crash-2022-10-08_20.55.06-server.txt)

warn(exemple) : 

```
[21:51:41] [Client thread/INFO] [STDOUT]: [net.tclproject.mysteriumlib.asm.core.MiscUtils$SystemLogHelper:warning:92]: [WARNING] Can not find the target method of fix: ASMFix: net.minecraft.world.biome.BiomeDecorator#func_150512_a(Lnet/minecraft/world/World;Ljava/util/Random;Lnet/minecraft/world/biome/BiomeGenBase;II) -> net.tclproject.mysteriumlib.asm.fixes.MysteriumPatchesFixesH#func_150512_a(Lnet/minecraft/world/biome/BiomeDecorator;Lnet/minecraft/world/World;Ljava/util/Random;Lnet/minecraft/world/biome/BiomeGenBase;II)Z, EnumReturnSetting=ON_TRUE, EnumReturnType=FIX_METHOD_RETURN_VALUE, InjectorFactory: net.tclproject.mysteriumlib.asm.core.FixInserterFactory$OnEnter, CreateMethod = false
```

logs : 

https://paste.ee/p/eybsQ

**Comment by quentin452** [comment]
temporary workaround fix, downgrade gasstation to 0.3.2 , warns is introduced in 0.3.3 ,but now the modpack is not working

maybe caused by https://github.com/FalsePattern/GasStation/commit/ee767edf506ca6fe3b23f07575294d1c1e3bd582 commit or https://github.com/FalsePattern/GasStation/commit/629ca36e31681466824b2918b349b4defbcf7bdf ?

need to wait a fix from gasstation

**Comment by quentin452** [comment]
fixed in v1.5
